component.js

```diff
@@ -102,6 +103,7 @@ function factory (options) {
   var _isJsx = !!options.jsx;
   var _hiddenCursorField = options.cursorField || '__singleCursor';
   var _isNode = options.isNode || isNode;
+  var _cached = cached.withDefaults(_shouldComponentUpdate);
```

Comment:
Just to be clear I understand, what you suggesting to do is:

```js
var _cached = cached.withDefaults(options);
```

And then in `cached` factory do:

```js
function factory (methods) {
  var isEqualProps = methods. isEqualProps || shouldUpdate.isEqualProps;
}
```

Is my understanding correct ?

Also I believe `isEqualProps` will work for the reason I've posted earlier.